### PR TITLE
Document repl new key binding(ctrl+s)

### PR DESCRIPTION
**Pull request by sigmaSd**
https://github.com/denoland/deno/pull/14536

### CODE REVIEW

File: tools/repl.md [hunk 1]
```diff
@@ -161,3 +161,4 @@ readTextFileSync  readFileSync      readDir           readLink          readAllS
 | Meta-T                | Transpose words                                                                                  |
 | Meta-U                | Upper-case the next word                                                                         |
 | Meta-Y                | See Ctrl-Y                                                                                       |
+| Alt-Enter             | Insert a new line                                                                                |
```
Comment (bartlomieju) [inline tools/repl.md:164]: This actually requires `Control` key on macOS

Comment (sigmaSd) [reply tools/repl.md:164]: There are other Key combo with alt, like alt left alt right, should those also mention Control on mac ?

Comment (bartlomieju) [reply tools/repl.md:164]: Okay, then please mention that it's `Control` on mac

Comment (sigmaSd) [reply tools/repl.md:164]: what do you think about adding a footer like this:
`Note: Alt key is replaced with Control on mac os`

Comment (sigmaSd) [reply tools/repl.md]: Updated the key to CTRL-S

**Review by bartlomieju — APPROVED** [review]
Thanks!

**Pull request merged**